packages/core/src/BaseSwitch/BaseSwitch.tsx

```diff
@@ -1,9 +1,11 @@
 import { forwardRef, useCallback, useState } from "react";
+import { css } from "@emotion/css";
```

Comment:
wrong `css`. also, let's please avoid this. no need for "runtime" styles when we have CSS vars & data attributes